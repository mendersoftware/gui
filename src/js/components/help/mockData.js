// Copyright 2020 Northern.tech AS
//
//    Licensed under the Apache License, Version 2.0 (the "License");
//    you may not use this file except in compliance with the License.
//    You may obtain a copy of the License at
//
//        http://www.apache.org/licenses/LICENSE-2.0
//
//    Unless required by applicable law or agreed to in writing, software
//    distributed under the License is distributed on an "AS IS" BASIS,
//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
//    See the License for the specific language governing permissions and
//    limitations under the License.
export const helpProps = {
  docsVersion: undefined,
  isHosted: false,
  menderVersion: 'master',
  isEnterprise: true,
  tenantCapabilities: {
    hasDeviceConfig: true,
    hasMonitor: true
  },
  tokens: [],
  versions: {
    repos: {
      mender: '3.3.0',
      'mender-artifact': '3.8.0',
      'mender-binary-delta': '1.3.1',
      'mender-cli': '1.8.0',
      'mender-configure-module': '1.0.4',
      'mender-connect': '2.0.2',
      'mender-convert': '3.0.0',
      'mender-gateway': '1.0.0',
      'monitor-client': '1.2.0'
    },
    releaseDate: '2003-01-01'
  }
};
