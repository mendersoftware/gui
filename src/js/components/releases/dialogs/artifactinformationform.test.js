import React from 'react';

import { undefineds } from '../../../../../tests/mockData';
import { render } from '../../../../../tests/setupTests';
import ArtifactInformationForm, { ReleaseTooltip } from './artifactinformationform';

describe('ArtifactInformationForm Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<ArtifactInformationForm activeStep={0} deviceTypes={[]} updateCreation={jest.fn} onboardingState={{ complete: false }} />);
    const view = baseElement.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});

describe('ReleaseTooltip Component', () => {
  it('renders correctly', async () => {
    const { baseElement } = render(<ReleaseTooltip />);
    const view = baseElement.firstChild.firstChild;
    expect(view).toMatchSnapshot();
    expect(view).toEqual(expect.not.stringMatching(undefineds));
  });
});
