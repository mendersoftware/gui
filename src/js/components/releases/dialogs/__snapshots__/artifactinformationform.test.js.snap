// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtifactInformationForm Component renders correctly 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
  width: 100%;
  margin-top: 18px;
  min-width: 240px;
}

.emotion-1 {
  color: rgba(0, 0, 0, 0.54);
  line-height: 1.4375em;
  font-family: Lato,sans-serif;
  font-weight: 400;
  font-size: 0.9286rem;
  padding: 0;
  position: relative;
  color: rgba(0, 0, 0, 0.54);
  display: block;
  transform-origin: top left;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  max-width: 133%;
  position: absolute;
  left: 0;
  top: 0;
  -webkit-transform: translate(0, -1.5px) scale(0.75);
  -moz-transform: translate(0, -1.5px) scale(0.75);
  -ms-transform: translate(0, -1.5px) scale(0.75);
  transform: translate(0, -1.5px) scale(0.75);
  -webkit-transition: color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,-webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  transition: color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
}

.emotion-1.Mui-focused {
  color: #337a87;
}

.emotion-1.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
}

.emotion-1.Mui-error {
  color: #ab1000;
}

.emotion-1.Mui-focused {
  color: #337a87;
}

.emotion-2 {
  line-height: 1.4375em;
  font-family: Lato,sans-serif;
  font-weight: 400;
  font-size: 0.9286rem;
  color: rgba(10, 10, 11, 0.78);
  box-sizing: border-box;
  position: relative;
  cursor: text;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
  position: relative;
}

.emotion-2.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
  cursor: default;
}

label+.emotion-2 {
  margin-top: 16px;
}

.emotion-2:after {
  border-bottom: 2px solid #337a87;
  left: 0;
  bottom: 0;
  content: "";
  position: absolute;
  right: 0;
  -webkit-transform: scaleX(0);
  -moz-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  transition: transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  pointer-events: none;
}

.emotion-2.Mui-focused:after {
  -webkit-transform: scaleX(1) translateX(0);
  -moz-transform: scaleX(1) translateX(0);
  -ms-transform: scaleX(1) translateX(0);
  transform: scaleX(1) translateX(0);
}

.emotion-2.Mui-error:before,
.emotion-2.Mui-error:after {
  border-bottom-color: #ab1000;
}

.emotion-2:before {
  border-bottom: 1px solid rgba(0, 0, 0, 0.42);
  left: 0;
  bottom: 0;
  content: "\\00a0";
  position: absolute;
  right: 0;
  -webkit-transition: border-bottom-color 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: border-bottom-color 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  pointer-events: none;
}

.emotion-2:hover:not(.Mui-disabled, .Mui-error):before {
  border-bottom: 2px solid rgba(10, 10, 11, 0.78);
}

@media (hover: none) {
  .emotion-2:hover:not(.Mui-disabled, .Mui-error):before {
    border-bottom: 1px solid rgba(0, 0, 0, 0.42);
  }
}

.emotion-2.Mui-disabled:before {
  border-bottom-style: dotted;
}

.emotion-2:before {
  border-bottom: 1px solid rgb(224, 224, 224);
}

.emotion-2:hover:not($disabled):before {
  border-bottom: 2px solid #337a87!important;
}

.emotion-2:after {
  border-bottom: 2px solid #337a87;
}

.emotion-3 {
  font: inherit;
  letter-spacing: inherit;
  color: currentColor;
  padding: 4px 0 5px;
  border: 0;
  box-sizing: content-box;
  background: none;
  height: 1.4375em;
  margin: 0;
  -webkit-tap-highlight-color: transparent;
  display: block;
  min-width: 0;
  width: 100%;
  -webkit-animation-name: mui-auto-fill-cancel;
  animation-name: mui-auto-fill-cancel;
  -webkit-animation-duration: 10ms;
  animation-duration: 10ms;
}

.emotion-3::-webkit-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-3::-moz-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-3:-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-3::-ms-input-placeholder {
  color: currentColor;
  opacity: 0.42;
  -webkit-transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: opacity 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-3:focus {
  outline: 0;
}

.emotion-3:invalid {
  box-shadow: none;
}

.emotion-3::-webkit-search-decoration {
  -webkit-appearance: none;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3::-webkit-input-placeholder {
  opacity: 0!important;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3::-moz-placeholder {
  opacity: 0!important;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:-ms-input-placeholder {
  opacity: 0!important;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3::-ms-input-placeholder {
  opacity: 0!important;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus::-webkit-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus::-moz-placeholder {
  opacity: 0.42;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus:-ms-input-placeholder {
  opacity: 0.42;
}

label[data-shrink=false]+.MuiInputBase-formControl .emotion-3:focus::-ms-input-placeholder {
  opacity: 0.42;
}

.emotion-3.Mui-disabled {
  opacity: 1;
  -webkit-text-fill-color: rgba(0, 0, 0, 0.38);
}

.emotion-3:-webkit-autofill {
  -webkit-animation-duration: 5000s;
  animation-duration: 5000s;
  -webkit-animation-name: mui-auto-fill;
  animation-name: mui-auto-fill;
}

.emotion-4 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  position: relative;
  min-width: 0;
  padding: 0;
  margin: 0;
  border: 0;
  vertical-align: top;
  margin-top: 18px;
  min-width: 240px;
}

.emotion-5 {
  color: rgba(0, 0, 0, 0.54);
  line-height: 1.4375em;
  font-family: Lato,sans-serif;
  font-weight: 400;
  font-size: 0.9286rem;
  padding: 0;
  position: relative;
  color: rgba(0, 0, 0, 0.54);
  display: block;
  transform-origin: top left;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  max-width: 100%;
  position: absolute;
  left: 0;
  top: 0;
  -webkit-transform: translate(0, 20px) scale(1);
  -moz-transform: translate(0, 20px) scale(1);
  -ms-transform: translate(0, 20px) scale(1);
  transform: translate(0, 20px) scale(1);
  -webkit-transition: color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,-webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  transition: color 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms,max-width 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
}

.emotion-5.Mui-focused {
  color: #337a87;
}

.emotion-5.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
}

.emotion-5.Mui-error {
  color: #ab1000;
}

.emotion-5.Mui-focused {
  color: #337a87;
}

.emotion-6 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.1607rem;
}

.emotion-6 iconButton {
  margin-right: 8px;
}

.emotion-7 {
  line-height: 1.4375em;
  font-family: Lato,sans-serif;
  font-weight: 400;
  font-size: 0.9286rem;
  color: rgba(10, 10, 11, 0.78);
  box-sizing: border-box;
  position: relative;
  cursor: text;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
}

.emotion-7.Mui-disabled {
  color: rgba(0, 0, 0, 0.38);
  cursor: default;
}

label+.emotion-7 {
  margin-top: 16px;
}

.emotion-7:after {
  border-bottom: 2px solid #337a87;
  left: 0;
  bottom: 0;
  content: "";
  position: absolute;
  right: 0;
  -webkit-transform: scaleX(0);
  -moz-transform: scaleX(0);
  -ms-transform: scaleX(0);
  transform: scaleX(0);
  -webkit-transition: -webkit-transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  transition: transform 200ms cubic-bezier(0.0, 0, 0.2, 1) 0ms;
  pointer-events: none;
}

.emotion-7.Mui-focused:after {
  -webkit-transform: scaleX(1) translateX(0);
  -moz-transform: scaleX(1) translateX(0);
  -ms-transform: scaleX(1) translateX(0);
  transform: scaleX(1) translateX(0);
}

.emotion-7.Mui-error:before,
.emotion-7.Mui-error:after {
  border-bottom-color: #ab1000;
}

.emotion-7:before {
  border-bottom: 1px solid rgba(0, 0, 0, 0.42);
  left: 0;
  bottom: 0;
  content: "\\00a0";
  position: absolute;
  right: 0;
  -webkit-transition: border-bottom-color 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: border-bottom-color 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  pointer-events: none;
}

.emotion-7:hover:not(.Mui-disabled, .Mui-error):before {
  border-bottom: 2px solid rgba(10, 10, 11, 0.78);
}

@media (hover: none) {
  .emotion-7:hover:not(.Mui-disabled, .Mui-error):before {
    border-bottom: 1px solid rgba(0, 0, 0, 0.42);
  }
}

.emotion-7.Mui-disabled:before {
  border-bottom-style: dotted;
}

.emotion-7:before {
  border-bottom: 1px solid rgb(224, 224, 224);
}

.emotion-7:hover:not($disabled):before {
  border-bottom: 2px solid #337a87!important;
}

.emotion-7:after {
  border-bottom: 2px solid #337a87;
}

.emotion-9.Mui-focused .MuiAutocomplete-clearIndicator {
  visibility: visible;
}

@media (pointer: fine) {
  .emotion-9:hover .MuiAutocomplete-clearIndicator {
    visibility: visible;
  }
}

.emotion-9 .MuiAutocomplete-tag {
  margin: 3px;
  max-width: calc(100% - 6px);
}

.emotion-9 .MuiAutocomplete-inputRoot {
  -webkit-box-flex-wrap: wrap;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
}

.MuiAutocomplete-hasPopupIcon.emotion-9 .MuiAutocomplete-inputRoot,
.MuiAutocomplete-hasClearIcon.emotion-9 .MuiAutocomplete-inputRoot {
  padding-right: 30px;
}

.MuiAutocomplete-hasPopupIcon.MuiAutocomplete-hasClearIcon.emotion-9 .MuiAutocomplete-inputRoot {
  padding-right: 56px;
}

.emotion-9 .MuiAutocomplete-inputRoot .MuiAutocomplete-input {
  width: 0;
  min-width: 30px;
}

.emotion-9 .MuiInput-root {
  padding-bottom: 1px;
}

.emotion-9 .MuiInput-root .MuiInput-input {
  padding: 4px 4px 4px 0px;
}

.emotion-9 .MuiInput-root.MuiInputBase-sizeSmall .MuiInput-input {
  padding: 2px 4px 3px 0;
}

.emotion-9 .MuiOutlinedInput-root {
  padding: 9px;
}

.MuiAutocomplete-hasPopupIcon.emotion-9 .MuiOutlinedInput-root,
.MuiAutocomplete-hasClearIcon.emotion-9 .MuiOutlinedInput-root {
  padding-right: 39px;
}

.MuiAutocomplete-hasPopupIcon.MuiAutocomplete-hasClearIcon.emotion-9 .MuiOutlinedInput-root {
  padding-right: 65px;
}

.emotion-9 .MuiOutlinedInput-root .MuiAutocomplete-input {
  padding: 7.5px 4px 7.5px 5px;
}

.emotion-9 .MuiOutlinedInput-root .MuiAutocomplete-endAdornment {
  right: 9px;
}

.emotion-9 .MuiOutlinedInput-root.MuiInputBase-sizeSmall {
  padding-top: 6px;
  padding-bottom: 6px;
  padding-left: 6px;
}

.emotion-9 .MuiOutlinedInput-root.MuiInputBase-sizeSmall .MuiAutocomplete-input {
  padding: 2.5px 4px 2.5px 8px;
}

.emotion-9 .MuiFilledInput-root {
  padding-top: 19px;
  padding-left: 8px;
}

.MuiAutocomplete-hasPopupIcon.emotion-9 .MuiFilledInput-root,
.MuiAutocomplete-hasClearIcon.emotion-9 .MuiFilledInput-root {
  padding-right: 39px;
}

.MuiAutocomplete-hasPopupIcon.MuiAutocomplete-hasClearIcon.emotion-9 .MuiFilledInput-root {
  padding-right: 65px;
}

.emotion-9 .MuiFilledInput-root .MuiFilledInput-input {
  padding: 7px 4px;
}

.emotion-9 .MuiFilledInput-root .MuiAutocomplete-endAdornment {
  right: 9px;
}

.emotion-9 .MuiFilledInput-root.MuiInputBase-sizeSmall {
  padding-bottom: 1px;
}

.emotion-9 .MuiFilledInput-root.MuiInputBase-sizeSmall .MuiFilledInput-input {
  padding: 2.5px 4px;
}

.emotion-9 .MuiInputBase-hiddenLabel {
  padding-top: 8px;
}

.emotion-9 .MuiFilledInput-root.MuiInputBase-hiddenLabel {
  padding-top: 0;
  padding-bottom: 0;
}

.emotion-9 .MuiFilledInput-root.MuiInputBase-hiddenLabel .MuiAutocomplete-input {
  padding-top: 16px;
  padding-bottom: 17px;
}

.emotion-9 .MuiFilledInput-root.MuiInputBase-hiddenLabel.MuiInputBase-sizeSmall .MuiAutocomplete-input {
  padding-top: 8px;
  padding-bottom: 9px;
}

.emotion-9 .MuiAutocomplete-input {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  text-overflow: ellipsis;
  opacity: 1;
}

<div>
  <div
    class="flexbox column"
  >
    <div />
    <div
      class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root emotion-0"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-standard MuiFormLabel-colorPrimary Mui-focused MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-standard emotion-1"
        data-shrink="true"
        for="mui-1"
        id="mui-1-label"
      >
        Destination directory where the file will be installed on your devices
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth Mui-focused MuiInputBase-formControl emotion-2"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input emotion-3"
          id="mui-1"
          placeholder="Example: /opt/installed-by-single-file"
          style="margin-top: 16px;"
          type="text"
          value=""
        />
      </div>
    </div>
    <h4>
      Artifact information
    </h4>
    <div
      class="MuiFormControl-root emotion-4"
    >
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard emotion-5"
        data-shrink="false"
        for="release-name"
        style="align-items: center; display: flex;"
      >
        Release name
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root margin-left-small MuiSvgIcon-fontSizeSmall emotion-6"
          data-mui-internal-clone-element="true"
          data-testid="InfoOutlinedIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
          />
        </svg>
      </label>
      <div
        class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl release-name-input emotion-7"
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiInput-input emotion-3"
          id="release-name"
          placeholder="A descriptive name for the software"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      class="MuiAutocomplete-root emotion-9"
    >
      <div
        class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root emotion-0"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-standard emotion-5"
          data-shrink="false"
          for="compatible-device-type-selection"
          id="compatible-device-type-selection-label"
        >
          Device types compatible
        </label>
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiAutocomplete-inputRoot emotion-2"
        >
          <input
            aria-autocomplete="list"
            aria-expanded="false"
            aria-invalid="false"
            autocapitalize="none"
            autocomplete="off"
            class="MuiInputBase-input MuiInput-input MuiAutocomplete-input MuiAutocomplete-inputFocused emotion-3"
            id="compatible-device-type-selection"
            placeholder="Enter all device types this software is compatible with"
            role="combobox"
            spellcheck="false"
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ReleaseTooltip Component renders correctly 1`] = `
<div
  style="font-size: 12px;"
>
  <p>
    If a Release with this name already exists, this new Artifact may be grouped into a Release with other Artifacts of the same name - so long as they are compatible with different device types
  </p>
  <a
    href="/help/releases-artifacts"
    style="color: rgb(103, 155, 165);"
  >
    Learn more about releases
  </a>
</div>
`;
